server/src/utils.js

```diff
@@ -0,0 +1,64 @@
+const {DataSource} = require('apollo-datasource');
+
+const createStore = () => {
+  // Fake DB
+  // mockup data
+  // todo: move this into separated CSV file
+  //  and import it into DB
```

Comment:
I think you can fit this in the line above, and still be under 80 chars. Pull up.

Reply:
Fixed it. will keep it in mind. Thank you. 😊